### ISSUE TITLE
Reword device channel list introduction

### ISSUE THREAD

**Issue by aacuevas** [open-ephys/ONI#46]
MUST seems wrong here since two of the following elements are optional?

_Originally posted by @jonnew in https://github.com/open-ephys/ONI/pull/45#discussion_r1847129021_

**Comment by jonnew** [comment]
We decided on the following path:

Instead of this:

```
A device MUST expose the following :ref:`communication channels
<com-channels>`:

- A REQUIRED register interface
- An OPTIONAL read stream
- An OPTIONAL write stream
```

use this:

```
A device exposes a set  :ref:`communication channels <com-channels>`:

- It MUST expose a register interface. 
- It MAY a read stream.
- It MAY expose a write stream.
```

Additionally, provide reference links for the thread communication channel types in the list

**Comment by aacuevas** [comment]
Regarding the last point, links will not be provided because we do not have specific sections for the device streams, the base definitions on comm-channels plus what it is explained on this pae and sample format is all that applies to them

**Comment by aacuevas** [comment]
Fixed in #63
